services/API-service/src/api/process-pipeline/process-pipeline.controller.ts

```diff
@@ -43,7 +43,7 @@ export class ProcessPipelineController {
     status: 201,
     description: 'Closed finished events.',
   })
-  @Post('events/close-events')
+  @Post('event/close-events')
```

Comment:
@jannisvisser I thought this was intentional.

Reply:
Well it was somewhat, in the sense of changing all paths in process-pipeline.controller to /events. I just didn't realize here that this endpoint was kept in for the transition period specifically and thus needed an unchanged api path.